Sample_v3/Samples/SimpleChat/SimpleChannelMembersViewController.swift

```diff
@@ -0,0 +1,201 @@
+                completion(error == nil)
+                _ = memberController
+            }
+            
+            if member.isBanned {
+                memberController.unban(completion: actionCompletion)
+            } else {
+                memberController.ban(completion: actionCompletion)
+            }
+        }
+        
+        let configuration = UISwipeActionsConfiguration(actions: [removeAction, banAction])
+        configuration.performsFirstActionWithFullSwipe = true
+        return configuration
+    }
+    
+    ///
+    /// # cellForRowAt
+    ///
+    /// The method below returns a cell configured based on the member in position `indexPath.row` of `memberListController.members`.
+    ///
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let member = memberListController.members[indexPath.row]
+        return memberCell(member, isCurrentUser: member.id == memberListController.client.currentUserId)
+    }
+    
+    // MARK: - UITableViewDelegate
+    
+    ///
+    /// # willDisplay
+    ///
+    /// The method below handles the case when the last cell in the members list is displayed
+    /// by calling `memberListController.loadNextMembers()` to fetch more channel members.
+    ///
+    override func tableView(_ tableView: UITableView, willDisplay cell: UITableViewCell, forRowAt indexPath: IndexPath) {
+        if indexPath.section == tableView.numberOfSections - 1,
+            indexPath.row == tableView.numberOfRows(inSection: indexPath.section) - 1 {
+            memberListController.loadNextMembers()
+        }
+    }
+    
+    // MARK: - UI Code
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        title = "Members"
+        
+        navigationItem.rightBarButtonItem = .init(
+            barButtonSystemItem: .add,
+            target: self,
+            action: #selector(addNewMember)
+        )
+        
+        tableView.tableFooterView = UIView()
+    }
+    
+    @objc private func addNewMember() {
+        guard
+            let usersViewController = UIStoryboard.simpleChat
+            .instantiateViewController(withIdentifier: "SimpleUsersViewController") as? SimpleUsersViewController
+        else { return }
+        
+        usersViewController.userListController = memberListController.client.userListController(
+            query: .init(sort: [.init(key: .lastActivityAt)])
+        )
+
+        usersViewController.openDirectMessagesChat = { [weak self] userId in
+            self?.dismiss(animated: true) {
+                guard let self = self else { return }
+                
+                let channelController = self.memberListController.client.channelController(for: self.memberListController.query.cid)
+                channelController.addMembers(userIds: [userId]) { [channelController] _ in
+                    _ = channelController
+                }
```

Comment:
I don't think this is correct. The direct message channel controller must be created by `channelController(createDirectMessageChannelWith members: Set<UserId>)`.

Reply:
We don't open direct messages here but add the selected user as a channel member.

You are confused because I forgot to rename `openDirectMessagesChat` -> `didSelectUser` in **SimpleUsersViewController**. 

Will do the rename, thanks for careful review!

Reply:
That was the reason :) Thanks